impl/src/rendering/templates/partials/common_attrs.template

```diff
@@ -20,9 +20,16 @@
     {%- if target.kind != "bin" %}
     crate_type = "{{ target.kind }}",
     {%- endif %}
-    {%- if crate.raze_settings.data_attr %}
-    data = {{crate.raze_settings.data_attr}},
-    {%- endif %}
+    data = []
+    {%- if crate.raze_settings.data_attr %} + {{crate.raze_settings.data_attr}}
+    {%- endif -%}
+    {# Note: data_dependencies can only be set when using cargo-raze as a library at the moment #}
```

Comment:
I feel like this comment would also be nice to have in `Context.rs` or something

Reply:
Sounds good, I added comments as suggested